### ISSUE TITLE
WSL2 Install error `version `GLIBC_2.33' not found`

### ISSUE THREAD

**Issue by jadonbrownza** [m1guelpf/plz-cli#16]
![image](https://user-images.githubusercontent.com/7773361/208615467-93bb73b1-7d12-42f4-a9d1-08c603829914.png)

**Comment by maximofn** [comment]
I have the same error

```
$ curl -fsSL https://raw.githubusercontent.com/m1guelpf/plz-cli/main/install.sh | sh -
sh: 26: [[: not found
https://github.com/m1guelpf/plz-cli/releases/latest/download/plz-linux-x86_64
downloading latest binary
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13.1M  100 13.1M    0     0   651k      0  0:00:20  0:00:20 --:--:--  917k
/home/maximo.fernandez@AEROESPACIAL.SENER/.bin/plz: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /home/maximo.fernandez@AEROESPACIAL.SENER/.bin/plz)
/home/maximo.fernandez@AEROESPACIAL.SENER/.bin/plz: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /home/maximo.fernandez@AEROESPACIAL.SENER/.bin/plz)
/home/maximo.fernandez@AEROESPACIAL.SENER/.bin/plz: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /home/maximo.fernandez@AEROESPACIAL.SENER/.bin/plz)
installed -
```

**Comment by pegre94** [comment]
Same error on Ubuntu 20.04.

**Comment by bjoern247** [comment]
Same error here

**Comment by WillPresley** [comment]
I was having this issue as well on WSL2/Ubuntu 20 LTS. Upgrading to 22 LTS took care of the issue, as it was ultimately about the wrong C libraries. I think this could be solved during compilation, but not sure how.

**Comment by thomastx05** [comment]
So the install is trying to use a version higher than what you all have... had the same issue. If you download and install rust and then cargo build on your system it should work.

so 
1. Install rust
2. clone repository
3. cargo build
4. cargo run

**Comment by jadonbrownza** [comment]
We've got to do this for each release?

**Comment by thomastx05** [comment]
> We've got to do this for each release?

I am not too familiar with rust, but the best I can tell is that the file is compiled using a certain version of libc6. If your system is not using the same version then you need to do this. Once it is built you can just put the compiled binary in your PATH (this is most of what the first part of the install script is doing anyway) and it will work like normal from then on as far as I can tell. That worked for me.

**Comment by CohleRustW** [comment]
> So the install is trying to use a version higher than what you all have... had the same issue. If you download and install rust and then cargo build on your system it should work.
> 
> so
> 
> 1. Install rust
> 2. clone repository
> 3. cargo build
> 4. cargo run

you can use ``cargo build --release`` command to build ``plz`` binary.

**Comment by m1guelpf** [comment]
This should have been fixed by #19 and the new tagged release, could someone download the `musl` binary and confirm?

**Comment by jadonbrownza** [comment]
Can confirm, running the curl command now installs plz 🙂 I can now provide a OpenAI key.
